Update about-github-pages.md (#36175)
github/docs: content/pages/getting-started-with-github-pages/about-github-pages.md

There are three types of {% data variables.product.prodname_pages %} sites: project, user, and organization. Project sites are connected to a specific project hosted on {% data variables.product.github %}, such as a JavaScript library or a recipe collection. User and organization sites are connected to a specific account on {% data variables.location.product_location %}.

> To publish a user site, you must create OR rename a repository owned by your personal account that's named {% ifversion fpt or ghec %}`<username>.github.io`{% else %}`<username>.<hostname>`{% endif %}. To publish an organization site, you must create OR rename a repository owned by an organization that's named {% ifversion fpt or ghec %}`<organization>.github.io`{% else %}`<organization>.<hostname>`{% endif %}. {% ifversion fpt or ghec %}Unless you're using a custom domain, user and organization sites are available at `http(s)://<username>.github.io` or `http(s)://<organization>.github.io`.{% endif %}

Reviewer: To best present this info at the point of need, I actually suggest simplifying this section to avoid mentioning creating or renaming at all. If you want, you could then edit [Creating a repository for your site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-a-repository-for-your-site) to explicitly mention the option of renaming a repo. How does that sound to you?

```suggestion
* User sites are hosted in a repository owned your personal account, named {% ifversion fpt or ghec %}`<username>.github.io`{% else %}`<username>.<hostname>`{% endif %}.
* Organization sites are hosted in a repository owned by an organization, named {% ifversion fpt or ghec %}`<organization>.github.io`{% else %}`<organization>.<hostname>`{% endif %}.

{% ifversion fpt or ghec %}Unless you're using a custom domain, user and organization sites are available at `http(s)://<username>.github.io` or `http(s)://<organization>.github.io`.{% endif %}
```

Author: > To best present this info at the point of need, I actually suggest simplifying this section to avoid mentioning creating or renaming at all. If you want, you could then edit [Creating a repository for your site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-a-repository-for-your-site) to explicitly mention the option of renaming a repo. How does that sound to you?

Great. Actually saying create OR rename was a bit of a strech. I don't really want to edit another thing, but this is exactly what I meant.

Author: Thanks a lot.

> 
> **Note: Github pages works around 10 minutes after you renamed/created the repository.**
> 

Reviewer: I checked and we actually have a piece of reusable content that should cover this 🙂 It would be good to use that here to help maintainability

```suggestion
{% data reusables.pages.twenty-minutes-to-publish %}
```

Despite the misleading name, it says:

> It can take up to 10 minutes for changes to your site to publish after you push the changes to {% data variables.product.github %}. If you don't see your {% data variables.product.prodname_pages %} site changes reflected in your browser after an hour, see [AUTOTITLE](/pages/setting-up-a-github-pages-site-with-jekyll/about-jekyll-build-errors-for-github-pages-sites).

Author: This is great. Thanks a lot
